scripts/predict_new_vids.py

```diff
@@ -61,7 +61,7 @@ def predict_videos_in_dir(cfg: DictConfig):
         absolute_cfg_path = return_absolute_path(hydra_relative_path, n_dirs_back=2)
 
         # load model
-        model_cfg = OmegaConf.load(os.path.join(absolute_cfg_path, ".hydra/config.yaml"))
+        model_cfg = OmegaConf.load(os.path.join(absolute_cfg_path, "config.yaml"))
```

Comment:
Another fix here, the original code loads the config without CLI overrides - hydra stores overrides separately in .hydra/overrides.yaml

Reply:
good catch!